From be91a981dfbc4ea9e6f1c47fafdec3132ca09923 Mon Sep 17 00:00:00 2001
From: Bram Verweij <amverweij@gmail.com>
Date: Wed, 8 Nov 2006 18:00:52 +0200
Subject: [PATCH] xfree86/linux acpi: fix tokenising

Split on a space, rather on the 'video' string, as strtok takes a char,
not a string.
(cherry picked from 0567a6337b84fa045b5732e98203f488274aa2a2 commit)
---
 hw/xfree86/os-support/linux/lnx_acpi.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Index: xorg-server/hw/xfree86/os-support/linux/lnx_acpi.c
===================================================================
--- xorg-server.orig/hw/xfree86/os-support/linux/lnx_acpi.c	2007-02-03 22:50:12.000000000 +0100
+++ xorg-server/hw/xfree86/os-support/linux/lnx_acpi.c	2007-02-03 22:51:10.000000000 +0100
@@ -65,7 +65,7 @@
 	char *data = NULL; /* doesn't appear to be used in the kernel */
 	unsigned long int notify_l, data_l;
 
-	video = strtok(ev, "video");
+	video = strtok(ev, " ");
 
 	GFX = strtok(NULL, " ");
 #if 0
